lib/Importer/ONNXIFIModelLoader.cpp

```diff
@@ -209,6 +209,8 @@ ONNXIFIModelLoader::parseOperators(const void *onnxModel,
     } else if (operation == "DotProduct") {
       ADD_OP_MAPPING(BatchedReduceAddNodeKind, FloatTy);
       ADD_OP_MAPPING(MulNodeKind, FloatTy);
+    } else if (operation == "LengthsToRanges") {
+      ADD_OP_MAPPING(LengthsToRangesNodeKind, Int64ITy);
```

Comment:
we need to add this to `isOpSupported` of CPU backend and Interpreter as well.

Reply:
Question here: it's already supported by default since it's not using Int8QTy.  Do you mean I should add quantized support?  (Which is basically a NOP, so maybe it already just works?)

Reply:
I did not realize that it'll be supported by default. Ignore this comment.